third_party/move/extensions/async/move-async-vm/Cargo.toml

```diff
@@ -14,7 +14,7 @@ better_any = { workspace = true }
 bytes = { workspace = true }
 itertools = { workspace = true }
 move-binary-format = { path = "../../../move-binary-format" }
-move-command-line-common = { path = "../../../move-command-line-common" }
+# move-command-line-common = { path = "../../../move-command-line-common" }
```

Comment:
typo?

Reply:
I commented out some async tests, and to make lint pass, had to remove imports from this crate. Kept it in toml though.

Reply:
removed